test/WebSites/TagHelpersWebSite/TagHelpers/BoldTagHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace TagHelpersWebSite.TagHelpers
+{
+    [TargetElement(Attributes = "bold")]
+    public class BoldTagHelper : TagHelper
+    {
+        public override int Order
```

Comment:
The `ConditionTagHelper` conflicts with this one for the footer portion of the test. If this `BoldTagHelper` doesn't run first then the `ConditionTagHelper` can't properly suppress the output.